coalib/bears/Bear.py

```diff
@@ -249,8 +249,7 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except:
-            self.warn("Bear {} failed to run. Take a look at debug messages"
-                      " (`-V`) for further information.".format(name))
+            self.warn(self.get_warning_msg())
```

Comment:
Seems like your bear hasn't the method `get_warning_msg` so you can't use like this. You should check where are the other methods used here